source/features/update-pr-from-base-branch.tsx

```diff
@@ -63,20 +63,41 @@ async function addButton(mergeBar: Element): Promise<void> {
 		return;
 	}
 
-	const mergeabilityRow = $optional('.branch-action-item:has(.merging-body)');
+	const mergeabilityRow = $optional([
+		'.branch-action-item:has(.merging-body)',
+		'[aria-label="Conflicts"] [class^="MergeBoxSectionHeader-module__wrapper"]',
+	]);
+
+	const isNewView = mergeBar.className.startsWith('MergeBox-module');
+
 	if (mergeabilityRow) {
+		if (isNewView) {
```

Comment:
FYI "new" just means "current", this applies to all code. Code is permanent, I don't want to have 2-year-old lines with the word "new". You can specify "react" or just mark the old one as "old" instead (preferable, if the new view is intended to completely replace the old one)

Reply:
Cc @kidonng since I saw the same in your PR